con-master/src/main/resources/application.properties

```diff
@@ -1,6 +1,8 @@
 spring.datasource.url=jdbc:mysql://localhost/concierge
 spring.datasource.username=root
-spring.datasource.password=1234
+spring.datasource.password=root
```

Comment:
You're staging `application.properties`. Your team should probably think about having another unstaged e.g. `application-dev.properties` file where you can add your custom settings, or agree about common database and filesystem settings.